django_prices_openexchangerates/templatetags/prices_multicurrency_i18n.py

```diff
@@ -8,18 +8,18 @@
 
 
 @register.filter
-def in_currency(amount, currency, **kwargs):
-    converted_amount = exchange_currency(amount, currency)
-    converted_amount = converted_amount.quantize('.01')
-    return converted_amount
+def in_currency(value, currency, **kwargs):
+    converted_value = exchange_currency(value, currency)
+    converted_value = converted_value.quantize('.01')
+    return converted_value
 
 
 @register.simple_tag
-def discount_amount_in_currency(discount, amount, currency):
-    amount = exchange_currency(amount, to_currency=currency)
-    discount_amount = exchange_currency(discount.amount, to_currency=currency)
-    discount.amount = discount_amount
-    return (amount | discount) - amount
+def discount_amount_in_currency(discount, price, currency):
```

Comment:
I think `base` is better than `price`. Not sure whether we should keep this order of arguments.

Reply:
The arguments here are mostly like that for consistency with `django-prices`'s `discount_amount_for(discount, price)` template tag.